wenet/bin/train_nst.py

```diff
@@ -0,0 +1,319 @@
+# Copyright (c) 2021 Mobvoi Inc. (authors: Binbin Zhang)
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import argparse
+import copy
+import logging
+import os
+import torch
+import torch.distributed as dist
+import torch.optim as optim
+import yaml
+from tensorboardX import SummaryWriter
+from torch.utils.data import DataLoader
+from wenet.dataset.dataset import Dataset
+from wenet.transformer.asr_model import init_asr_model
```

Comment:
What's your base commits? we have refactor our model initialization in this PR https://github.com/wenet-e2e/wenet/pull/1280, call `wenet.utils.init_model.init_model` instead of `wenet.transformer.asr_model.init_asr_model`